scripts/build-image.ps1

```diff
@@ -0,0 +1,46 @@
+<#
```

Comment:
any reason why this ins't a mage target? just curious, I have 0 context around wins

Reply:
Wins doesn't have any docker utils that can be used by our mage files, and adding them may be a waste of time depending on how we want to standardize CI across all of the windows repos. So, yes it could be a mage target but for the sake of time and to avoid some of that work being undone in the future I opted for a simple powershell script instead.